### DISCUSSION TITLE
Problem with pre sense after patch. Any solution or do I have to restore

### DISCUSSION THREAD

**Discussion by traviesojaen83** [Mr-MIBonk/M.I.B._More-Incredible-Bash#513] [Q&A]
Hello everyone, I have had problems after patching my mib, specifically it is a MHI2Q_ER_AUG_P5092. The thing is that it used a patch, and I think that as a result of this the pre sense system has stopped working for me. The question is, is there a way to recover the pre sense system, or on the contrary, what option should be chosen to restore the unit to its state before the patch. On the other hand, do you know if the maps can be downgraded, since my version only supported until 2020 and I have them updated to the latest version. Thanks for the help in advance

**Comment by hans19210** [comment]
Hi

Did you manage to solve your problem? I have the same issue! 😒

**Comment by traviesojaen83** [comment]
Hello hans, I have already solved the problem. I run an adaptation reset to factory, in the module, from the module of mib with vagcom. This erased all related faults, and the pre sense worked again, carefully testing and driving
